src/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```

Comment:
Since you have React version 18, it's not necessary to import React it's better to remove it.

Reply:
When I remove it from the index file, I always get an error